testing/matrix-sdk-test/src/lib.rs

```diff
@@ -93,7 +93,7 @@ macro_rules! init_tracing_for_tests {
                     // Since tracing_subscriber does prefix matching, the `matrix_sdk=` directive
                     // takes effect for all the main crates (`matrix_sdk_base`, `matrix_sdk_crypto`
                     // and so on).
-                    "info,matrix_sdk=debug".into()
+                    "info,wiremock=debug,matrix_sdk=debug".into()
```

Comment:
Can we revert this change? wiremock can be verbose in the output…

Reply:
Ah right sorry. I had enabled it locally to understand why an event wasn't matching for an endpoint.